graph-db/src/utils/indexing/indexRelatedUrl.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Given a RelatedUrl object, Gremlin connection, and
+ * associated collection, index related URL and build
+ * relationships for any GENERAL DOCUMENTATION that exists
+ * in common with other nodes in the graph database
+ * @param {JSON} relatedUrl a list of RelatedUrls
+ * @param {Connection} gremlin a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+exports.indexRelatedUrl = (relatedUrl, gremlin, dataset) => {
+  const {
+    Type: type,
+    SubType: subType,
```

Comment:
the umm_json collection CMR response for related URLs has Subtype where the t is lowercase and not uppercase.

Reply:
I was using data in UAT while I was writing/testing this. Is that data not reliable?

Reply:
It should be, can you please provide me a collection where the related url contains a schema element of SubType and not Subtype?